test-configs.yaml

```diff
@@ -170,17 +170,233 @@ test_plans:
     params:
       job_timeout: '120'
 
-  ltp-ipc:
+  ltp-opts: &ltp-rootfs
     rootfs: debian_buster-ltp_nfs
     pattern: 'ltp/{category}-{method}-{protocol}-{rootfs}-ltp-template.jinja2'
     filters:
       - blocklist: *kselftest_defconfig_filter
-    params:
+    params: &ltp-params
       skip_install: "true"
-      tst_cmdfiles: "ipc"
+      tst_cmdfiles: "ltp"
```

Comment:
Maybe use an empty string here with a comment rather than `ltp` as it's not a real subset name.